client/src/pages/WeeklyTable.js

```diff
@@ -34,9 +35,9 @@ const WeeklyTable = ({ bookings, rowsCount }) => {
 					<tr>
 						{week.map((date) => (
 							<th key={date}>
-								{new Date(date).toLocaleDateString()}{" "}
+								{moment(date).format("ddd MMM Do")}{" "}
 								<ModalBookingScreen
-									bookingDate={new Date(date).toLocaleDateString()}
+									bookingDate={moment(date).format("ddd MMM Do")}
```

Comment:
I'm almost certain this prevents the BookingScreen from being able to save bookings. So I'm surprised you don't see the same problem?